src/indra_cogex/sources/pubmed/__init__.py

```diff
@@ -128,7 +217,7 @@ def _dump_edges(self) -> Path:
         return edges_path
 
 
-def get_url_paths(url: str) -> Generator:
+def get_url_paths(url: str) -> Generator[str, None, None]:
```

Comment:
why not `Iteratable[str]` which is easier to understand

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/139/commits/4831644ea419412820e5745c782ff0f87c0d8f6e